### DISCUSSION TITLE
I want to start contributing to open source projects

### DISCUSSION THREAD

**Discussion by AVI-SHUKLA1110** [Cyfrin/foundry-full-course-cu#925] [Q&A]
Hi everyone ! I have been learning solidity for quite a time now and I want to now improve my knowledge by contributing to open source projects involving solidity codes, so can anyone please provide me the link of some good open source projects about the same ?

**Comment by theirrationalone** [comment]
You can contribute here also. 
You can contribute at foundry-devops. 
You can contribute at smartcontractkit 
You can contribute at codehawks.
